cpp/oneapi/dal/algo/basic_statistics/backend/gpu/compute_kernel_dense_misc.hpp

```diff
@@ -0,0 +1,213 @@
+/*******************************************************************************
+* Copyright 2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include "oneapi/dal/backend/primitives/utils.hpp"
+#include "oneapi/dal/util/common.hpp"
+
+#ifdef ONEDAL_DATA_PARALLEL
+
+namespace oneapi::dal::basic_statistics::backend {
+
+namespace de = dal::detail;
+namespace be = dal::backend;
+namespace pr = dal::backend::primitives;
+
+enum class bs_list : std::uint64_t {
+    max = 1UL << 0,
+    min = 1UL << 1,
+    sum = 1UL << 2,
+    sum2 = 1UL << 3,
+    sum2cent = 1UL << 4,
+    mean = 1UL << 5,
+    sorm = 1UL << 6,
+    varc = 1UL << 7,
+    stdev = 1UL << 8,
+    vart = 1UL << 9
```

Comment:
```suggestion
    max = 0b0,
    min = 0b10,
    sum = 0b100,
    sum2 = 0b1000,
    sum2cent = 0b10000,
    mean = 0b100000,
    sorm = 0b1000000,
    varc = 0b10000000,
    stdev = 0b100000000,
    vart = 0b1000000000
```

Reply:
What does abbreviation `sorm` mean? Should it be `snorm` or `square_norm` or what?

Reply:
sorm - second order raw moment, 
1UL << n - do you see any issue with this approach? It is more suitable from maintaining and visibility perspective.